venues/ICLR.cc/2019/process/submissionProcess.js

```diff
@@ -0,0 +1,87 @@
+function() {
+  var or3client = lib.or3client;
+  console.log('submission process');
```

Comment:
Could you prepare a submission process with only email notifications?

Reply:
I think @mohituniyal has that file already. he could probably just commit it to this branch

Reply:
I have committed the file. @mspector and @melisabok  Please take a look as I do not have a 100% confidence in the change.

Reply:
I would keep both versions, one that creates the blinded notes for testing purposes and the other one that only send emails, no revision invitation, for the live site.